src/library/Avatar/__tests__/Avatar.spec.js

```diff
@@ -4,6 +4,7 @@ import { shallow } from 'enzyme';
 import Avatar from '../Avatar';
 import examples from '../../../website/app/demos/Avatar/examples';
 import testDemoExamples from '../../../../utils/testDemoExamples';
+import testThemeOverrides from '../../../../utils/testThemeOverrides';
```

Comment:
It'd be nice, but not a blocker, to import these like `import { testDemoExamples, testThemeOverrides } from '../../../../utils/testing;` (filename at your discretion).